### PR TITLE
Update krnlmon reference point and fix build issues

### PR DESCRIPTION
**Pull request by nupuruttarwar**
Update krnlmon reference point to get recent bug fixes. Fix build issues by including daemon library sources

Signed-off-by: Nupur Uttarwar <nupur.uttarwar@intel.com>

### CODE REVIEW

File: infrap4d/CMakeLists.txt [hunk 1]
```diff
@@ -19,6 +19,8 @@ endif()
 
 set_install_rpath(infrap4d $ORIGIN/../lib ${SDE_ELEMENT} ${DEP_ELEMENT})
 
+target_sources(infrap4d PRIVATE $<TARGET_OBJECTS:daemon_o>)
```
Comment (ffoulkes) [inline infrap4d/CMakeLists.txt:22]: It wasn't omitted. There was a merge conflict with
```
set_install_rpath(infrap4d $ORIGIN/../lib ${SDE_ELEMENT} ${DEP_ELEMENT})
```
which I resolved in favor of `set_install_path`

**Review by ffoulkes — CHANGES_REQUESTED** [review]
It wasn't omitted. There was a merge conflict with a `

File: infrap4d/CMakeLists.txt (hunk 1 above)
Comment (ffoulkes) [reply infrap4d/CMakeLists.txt:22]: Wait a minute ... did I delete the wrong line?

**Review by ffoulkes — APPROVED** [review]
My bad. Serves me right for reading with less than my full attention.

File: infrap4d/CMakeLists.txt (hunk 1 above)
Comment (nupuruttarwar) [reply infrap4d/CMakeLists.txt:22]: Ohh okay. I think so. I don't see target_sources line in latest code

**Pull request merged**